app/assets/stylesheets/agents.scss

```diff
@@ -126,3 +126,10 @@
 .agent-status-color-red {
   background-color: #f33;
 }
+
+.lunch {
+  background-image: url("lunch.svg");
```

Comment:
Use `image-url` instead of plain `url` to include the image in Rails asset pipeline and thus work properly in production.